index.js

```diff
@@ -31,6 +31,10 @@ module.exports = input => {
 		throw new Error('You should not publish when running Node.js 6. Please downgrade and publish again. https://github.com/npm/npm/issues/5082');
 	}
 
+	if (!options.anyBranch && execa.sync('git', ['symbolic-ref', '--short', 'HEAD']).stdout !== 'master') {
```

Comment:
Heh whoops. Thanks for catching that. Using `Example` as precedence, I went with singular `Option` in the usage printout, since there's only one. If that's no good, happy to change it.